### ISSUE TITLE
Streaming not loading after following instructions for Brave Browser 

### ISSUE THREAD

**Issue by LucienLachance89** [pixeltris/TwitchAdSolutions#135]
I am currently using Brave Browser 

I installed uninstalled and reinstalled " Ublock Origin "  went and added " **twitch.tv##+js(twitch-videoad)** " to the filter list then went and turned on developer mode clicked on the settings  and added " **https://raw.githubusercontent.com/pixeltris/TwitchAdSolutions/bc355dda0ae98e142577e8521ad1857ccdb3b724/video-swap-new/video-swap-new-ublock-origin.js**" to the userreourceloaction   and then closed brave and re opened it . 

I then clicked on the   TTV LOL PRO   scrolled down and followed the directions to install on chrome  did everything it asked and the as soon as I did the registery key add and restarted my brave browser and went to a random stream to see if I get adds anymore the  stream no longer loads shows a inifinite buffering circle but the chat loads fine .  So the only way to get the player to actually work is by removing that reg edit key but the moment you do the stream will work but you will see a ad page pop up saying ads are being blocked please wait

**Comment by Thelgow** [comment]
I am experiencing similar on Firefox. I added the userResourcesLocation setting, "https://raw.githubusercontent.com/pixeltris/TwitchAdSolutions/bc355dda0ae98e142577e8521ad1857ccdb3b724/video-swap-new/video-swap-new-ublock-origin.js" and now 95% of the tabs just load forever. I get chat up fine, but a black screen with a loading circle. Out of curiosity I whitelisted that channel in TTVLOL and the stream came up immediately, no ads. I then Unwhitelisted it, screen refreshed and it was good. About 5 minutes later checking, 2 of 3 tabs did finally come up, yet 1 is still loading, approx 10 mins as I tabbed back over to confirm and now its running. 
I had TTVLOL Pro setup a couple weeks ago and it appeared fine until this morning, when I saw the recommendation to add the userResourcesLocation so it seems safe to narrow it down to that piece. 
Its just very random. I just opened 3 more tabs and 1 came up immediately other 2 are still loading after 5 mins. I remove the userResourcesLocation setting, hit F5 and both of those tabs came up fine.
